lib/matplotlib/finance.py

```diff
@@ -17,17 +17,20 @@
 from six.moves.urllib.request import urlopen
 
 import datetime
+import logging
```

Comment:
As this file is deprecated, you may want to send these changes to the new upstream.

Reply:
Not sure what you mean?  Whatever is replacing this file?  Not sure what that is!

Reply:
Oh `mpl_finance`?  Hmmm